core/toolbox.js

```diff
@@ -286,7 +312,7 @@ Blockly.Toolbox.prototype.getClientRect = function() {
     return new goog.math.Rect(-BIG_NUM, -BIG_NUM, 2 * BIG_NUM,
         BIG_NUM + y + height);
   } else {  // Bottom
-    return new goog.math.Rect(0, y, 2 * BIG_NUM, BIG_NUM + width);
+    return new goog.math.Rect(0, y - height, 2 * BIG_NUM, BIG_NUM);
```

Comment:
I agree that the + width is unnecessary, but subtracting height doesn't work here. 

Scribble skills on point:

![image](https://user-images.githubusercontent.com/13686399/30669289-a3a8b848-9e12-11e7-967b-c460ac34fb13.png)